app/index.html

```diff
@@ -1,222 +1,179 @@
 <html>
-
-<header>
+  <header>
     <link rel="stylesheet" href="grid.css" />
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
-        crossorigin="anonymous">
-</header>
+    <link
```

Comment:
This is very weird, should be in one line 🤔

Reply:
Discussed! Thanks!